test/integration/tcp_proxy_integration_test.cc

```diff
@@ -0,0 +1,125 @@
+#include "test/integration/tcp_proxy_integration_test.h"
+
+#include "common/network/utility.h"
+#include "common/ssl/context_manager_impl.h"
+
+#include "test/integration/ssl_utility.h"
+#include "test/integration/utility.h"
+#include "test/mocks/runtime/mocks.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+
```

Comment:
I have this longstanding wish to have all tests in the anonymous namespace, but that's just an observation, I don't even do this consistently today.